lib/segments/gt1.rb

```diff
@@ -0,0 +1,59 @@
+module HL7
+  class Message::Segment::GT1 < HL7::Message::Segment
+    add_field :set_id
+    add_field :guarantor_number
+    add_field :guarantor_name
+    add_field :guarantor_spouse_name
+    add_field :guarantor_address
+    add_field :guarantor_home_phone
+    add_field :guarantor_business_phone
+    add_field :guarantor_date_of_birth
+    add_field :guarantor_sex
+    add_field :guarantor_type
+    add_field :guarantor_relationship
+    add_field :guarantor_ssn
+    add_field :guarantor_begin_date
```

Comment:
Be worth running these dates through `covert_to_ts` to standardize the formatting.  https://github.com/ruby-hl7/ruby-hl7/blob/master/lib/segments/pid.rb#L47